package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -348,6 +349,10 @@ dhcp_host_add() {
 		# one (virtual) machine gets one DUID per RFC3315
 		duids="id:${duid// */}"
 	fi
+	
```

Comment:
This line contains white spaces; can you remove it ?

Reply:
My fault. Removed.